src/core/ee/vu.hpp

```diff
@@ -108,6 +112,7 @@ class VectorUnit
         void addq(uint8_t field, uint8_t dest, uint8_t source);
         void clip(uint8_t reg1, uint8_t reg2);
         void div(uint8_t ftf, uint8_t fsf, uint8_t reg1, uint8_t reg2);
+        void waitq();
```

Comment:
Place this in alphabetical order with the rest of the functions, for consistency.